nemo/collections/nlp/data/machine_translation/preproc_mt_data.py

```diff
@@ -147,6 +179,10 @@ def __init__(self, cfg: MTEncDecModelConfig, trainer: Trainer = None) -> None:
                         raise ValueError(
                             'src_file_name and tgt_file_name needed to create tarred dataset but could not be found.'
                         )
+                    if cfg.get('multilingual'):
+                        raise ValueError(
+                            'Tarred dataset cannot be created with multilingual tag set to True. Pre-process each dataset one-by-one.'
```

Comment:
Can't we loop through the datasets and process them one-by-one automatically?

Reply:
Actually, we can... Will add this in the next commit.

Reply:
Addressed this in the update.